datafaker-gen-examples/datafaker-gen-bigquery/pom.xml

```diff
@@ -147,14 +147,26 @@
                 <configuration>
                     <useManifestOnlyJar>false</useManifestOnlyJar>
                     <argLine>${surefire.argline}</argLine>
-                    <skipTests>${skipBigQueryTests}</skipTests>
+                    <skipTests>${skipBigQueryIntegrationTests}</skipTests>
```

Comment:
hm... this should be for `integration-test`  phase IIUC
or did I miss anything?